plugins/redis/plugin.spec.yaml

```diff
@@ -44,57 +68,66 @@ actions:
         type: string
         description: Key to delete
         required: true
+        example: example:1234:session
     output:
       count:
         title: Count
         type: integer
         description: Number of keys deleted
         required: false
+        example: 1
   set:
     title: Set
-    description: Set a key
+    description: This action is used to set a key to a string value. There is an optional expiration timeout which will auto remove the key when `expire` seconds have passed
     input:
       key:
         title: Key
         type: string
         description: Key to set
         required: true
+        example: example:1234:session
       value:
         title: Value
         type: string
         description: Value to set
         required: true
+        example: active
       expire:
         title: Expire
         type: integer
         description: Expiration in seconds
         required: false
+        example: 100
     output:
       reply:
         title: Reply
         type: string
         description: Reply (usually OK)
         required: false
+        example: OK
   get:
     title: Get
-    description: Get a key
+    description: This action is used to get a key. Get will return a value at `key` if found, otherwise found will be false
```

Comment:
```suggestion
    description: This action is used to get a key. It will return the key at `value` if found, otherwise found will be false.
```

Reply:
Changed it, but leaving a full stop at the end of the description causes the validator to flag. 😄